Dockerfile

```
@@ -11,43 +11,37 @@ USER 0
 COPY package.json yarn.lock lerna.json tsconfig.json .prettierrc ./
 COPY apps ./apps
 COPY libs ./libs
-RUN chmod 0500 -R apps libs
 RUN apk add python3 make g++
 RUN sed -i s^https://registry.yarnpkg.com^$YARNREPO^g yarn.lock
 RUN yarn --frozen-lockfile --production --network-timeout 600000
```

```suggestion
RUN yarn install --frozen-lockfile --production --network-timeout 600000
```

would feel better if we explicitly labeled the action being done here

---

no like, just make it a 'yarn install' instead of a 'yarn' - see the commit sugggestion i've provided.  it doesn't need a comment.

---

Missed that on the codespace GUI - yeah, I second that, because I didn't initially know what that line was doing, either.